src/midcom/datamanager/storage/parameter.php

```diff
@@ -10,12 +10,29 @@
  */
 class parameter extends delayed
 {
+    private $multiple_separator = '|';
+
+    /**
+     * This member contains the other key, in case it is set. In case of multiselects,
+     * the full list of unknown keys is collected here, in case of single select, this value
+     * takes precedence from the standard selection.
+     *
+     * This is only valid if the allow_other flag is set.
+     * TODO: still to be implentend
+     * @var string
+     */
+    private $other = array();
     /**
      * {@inheritdoc}
      */
     public function load()
     {
-        return $this->object->get_parameter($this->config['storage']['domain'], $this->config['storage']['name']);
+        $source = $this->object->get_parameter($this->config['storage']['domain'], $this->config['storage']['name']);
+        if(isset($this->config['type_config']['multiple_storagemode'])) {
```

Comment:
also note that `isset` will return `true` if `$this->config['type_config']['multiple_storagemode']` is set to `false` (it will only return `false` if the value is not set or `null`), so this needs to be an `empty` check instead